classes/genre.rb

```diff
@@ -0,0 +1,13 @@
+class Genre
+  attr_accessor :items, :genre
+
+  def initialize(name)
+    @name = name
+    @items = []
```

Comment:
- [ ] The app is not saving the genres. It shows an empty value when I select the options to list all genres. If you have some specific format please include it in the app.

Reply:
Thank you @eri8-9. The bug is fixed